.travis.yml

```diff
@@ -52,6 +52,7 @@ env:
         - JHI_DISABLE_WEBPACK_LOGS=true
         - JHI_E2E_HEADLESS=true
         - JHI_SCRIPTS=$TRAVIS_BUILD_DIR/test-integration/scripts
+        - NG_CLI_ANALYTICS="false"
```

Comment:
@vishal423 : can you explain me what is it plz ? Maybe we need to add this to Azure Pipelines too ?

Reply:
Angular CLI added an analytics feature (like us), this env variable prevents from sending data usage to the CLI project.
I don't think we need to add it into our pipeline as we don't rely on the angular CLI to build our application.

Reply:
In absence of this option, Travis builds got stuck in the post-install scripts (failed build logs @ https://travis-ci.org/jhipster/generator-jhipster/builds/539685496). I haven't noticed this issue on the Azure build pipleines (not sure why such discrepancy).